src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -64,6 +69,8 @@
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_HEIGHT_DESC = " " + PREFIX_HEIGHT + "17o"; // 'o' not allowed in height
+    // 2 decimal places is not allowed in weight
+    public static final String INVALID_WEIGHT_DESC = " " + PREFIX_WEIGHT + "55.55";
```

Comment:
Shouldn't comment be at the end just like all the other fields?

Reply:
I have tried appending the commend to the end of the line, but the line will exceed 120 chars. Hence, I moved it to the top.